intelmq/bots/experts/sieve/expert.py

```diff
@@ -85,8 +85,27 @@ def check(parameters):
             harmonization_config = utils.load_configuration(HARMONIZATION_CONF_FILE)
             SieveExpertBot.harmonization = harmonization_config['event']
 
-            metamodel = SieveExpertBot.init_metamodel()
-            SieveExpertBot.read_sieve_file(parameters['file'], metamodel)
+            grammarfile = os.path.join(os.path.dirname(__file__), 'sieve.tx')
+            if not os.path.exists(grammarfile):
+                raise FileExistsError('Sieve grammar file not found: %r.' % grammarfile)
+
+            metamodel = None
+
+            try:
+                metamodel = metamodel_from_file(grammarfile)
+            except TextXError as e:
+                raise TextXSemanticError('Could not process sieve grammar file. Error in (%d, %d).' % (e.line, e.col))
+
+            if not metamodel:
+                raise TextXError('Metamodel not loaded.')
+
+            if not os.path.exists(parameters['file']):
+                raise FileExistsError('File does not exist: %r' % parameters['file'])
```

Comment:
`FileExistsError`? It does not exist. We can use `intelmq.lib.exceptions.ConfigurationError` instead.